src/IOST/Account.h

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../PublicKey.h"
+#include "../PrivateKey.h"
+#include "../proto/IOST.pb.h"
+
+#include <stdint.h>
+#include <string>
+
+namespace TW::IOST {
+
+class Account {
+  public:
+    static bool isValid(const std::string& name);
+    static std::string encodePubKey(const PublicKey& publicKey);
+    Account(const std::string& name): name(name) {}
```

Comment:
See clang-tidy warnings: https://dev.azure.com/TrustWallet/Trust%20Wallet%20Core/_build/results?buildId=2388
```suggestion
    Account(std::string name): name(std::move(name)) {}
```